README.md

```diff
@@ -206,88 +204,91 @@ in =include/riak_pipe.hrl=.
-falls, since the =chashfun= function always produces "0", regardless
-of input.
-
-Using the value =follow= in the chashfun field means inputs for the
-fitting should be sent to the same vnode that generated them.  This is
-useful for maintaining data locality for a series of operations,
-instead of potentially pushing each modification's output across
-inter-node links.
-
-The second argument to =riak_pipe:exec/2= is a proplist of options for
-the pipeline as a whole.  Currently supported options are:
-
- - sink :: =fitting()= | *=undefined=*
-
-           Where workers for the final fitting should send messages.
-           Leave this =undefined= to deliver outputs as messages to
-           the client, the process that called =riak_pipe:exec/2=.
-
- - log :: =sink= | =lager= | =sasl= | *=undefined=*
-
-          If set to =sink=, log messages will be sent to the sink
-          process (just as outputs from the final fitting are).  If
-          set to =lager=, log messages are printed to the Riak log on
-          whatever node produces them.  If set to =sasl=, log messages
-          are printed to the SASL log on whatever node produces them.
-          If left =undefined=, log messages will be ignored silently.
-
- - trace :: =all= | =[trace_item()]= | *=undefined=*
-
-            If set to =all=, send all trace messages produced to the
-            log.  If set to a list, only send messages to the log if
-            one of their types matches one of the types listed.  If
-            left undefined, all trace messages are ignored silently.
-
-The =riak_pipe:exec/2= function will return a tuple of the form ={ok,
-Pipe}=, =Pipe= being a handle to the pipeline that was created (in the
-form of a =pipe= record, as defined in =include/riak_pipe.hrl=), which
+appear in error, log, and result messages generated by this fitting).
+The workers spawned would all run the `riak_pipe_w_pass` module against
+their inputs (see the <span class="spurious-link"
+target="Behavior">*Behavior*</span> section of this document for more
+examples of fitting worker behaviors included with `riak_pipe`).
+Finally, there would be only one worker spawned for the entire cluster,
+on the partition owning range in which the hash "0" falls, since the
+`chashfun` function always produces "0", regardless of input.
+
+Using the value `follow` in the chashfun field means inputs for the
+fitting should be sent to the same vnode that generated them. This is
+useful for maintaining data locality for a series of operations, instead
+of potentially pushing each modification's output across inter-node
+links.
+
+The second argument to `riak_pipe:exec/2` is a proplist of options for
+the pipeline as a whole. Currently supported options are:
+
+sink
+`fitting()` \| **`undefined`**
+
+Where workers for the final fitting should send messages. Leave this
+`undefined` to deliver outputs as messages to the client, the process
+that called `riak_pipe:exec/2`.
```

Comment:
The org-mode rendered this content as a definition list (`<dl><dt>sink</dt><dd>fitting()...</dd></dl>`). I can't tell if Github-flavored Markdown supports [definition lists](https://www.markdownguide.org/extended-syntax/#definition-lists). If it does, I think this would render better like

```
sink
: `figging()` \| **`undefined`**
: Where workers for the final fitting should...
```

If GFM does not support definition lists, then either a level-4 headers, or a bulleted list would help the structure of the definition stick out a little more, I think.

Reply:
I was using Pandoc to convert the formatting automatically, I will look into it.

Reply:
Definition list not supported, using h4 instead.